packages/web-react/src/components/Pagination/README.md

```diff
@@ -132,6 +144,16 @@ This component extends the `PaginationButtonLink` component.
 | -------------------- | -------- | ------- | -------- | ------------------------------- |
 | `accessibilityLabel` | `string` | `Next`  | ✕        | Accessibility label of the link |
 
+## UncontrolledPagination props
+
+| Name                 | Type                           | Default | Required | Description                                |
+| -------------------- | ------------------------------ | ------- | -------- | ------------------------------------------ |
+| `accessibilityLabel` | `string`                       | ✕       | ✕        | Accessibility label of the link            |
```

Comment:
what link? The arrows or the numbers? Am I able to set the a11y label of arrows?

Reply:
of the previous and next link...